### PR TITLE
feat: Added empty view for events fragment

### LINKED ISSUE #1141: Adding empty view for EventsFragment
**Describe the feature you'd like**

Adding an empty view indicating that there are no events available in the EventsFragment

**Screenshot**

**Would you like to work on the issue?**

<b>Yes</b>

### PR DESCRIPTION
**Pull request by anhanh11001**
Fixes #1141 Adding empty view for EventsFragment

Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt [hunk 1]
```diff
@@ -15,12 +15,7 @@ import com.google.android.material.snackbar.Snackbar
 import kotlinx.android.synthetic.main.content_no_internet.view.noInternetCard
 import kotlinx.android.synthetic.main.content_no_internet.view.retry
 import kotlinx.android.synthetic.main.fragment_events.eventsNestedScrollView
-import kotlinx.android.synthetic.main.fragment_events.view.eventsRecycler
-import kotlinx.android.synthetic.main.fragment_events.view.homeScreenLL
-import kotlinx.android.synthetic.main.fragment_events.view.locationTextView
-import kotlinx.android.synthetic.main.fragment_events.view.progressBar
-import kotlinx.android.synthetic.main.fragment_events.view.shimmerEvents
-import kotlinx.android.synthetic.main.fragment_events.view.swiperefresh
+import kotlinx.android.synthetic.main.fragment_events.view.*
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: Avoid wildcard imports.

File: app/src/main/res/layout/fragment_events.xml [hunk 2]
```diff
@@ -87,7 +87,13 @@
...
                         android:layout_width="match_parent"
                         android:layout_height="match_parent">
-
+                        <TextView
+                            android:id="@+id/noEventsMessage"
+                            android:layout_width="match_parent"
+                            android:layout_height="450dp"
+                            android:text="@string/no_events_message"
+                            android:gravity="center"
+                            />
```
Comment (liveHarshit) [inline app/src/main/res/layout/fragment_events.xml]: Newline not required.

File: app/src/main/res/layout/fragment_events.xml [hunk 3]
```diff
@@ -87,7 +87,13 @@
                     <FrameLayout
                         android:layout_width="match_parent"
                         android:layout_height="match_parent">
-
+                        <TextView
+                            android:id="@+id/noEventsMessage"
+                            android:layout_width="match_parent"
+                            android:layout_height="450dp"
```
Comment (liveHarshit) [inline app/src/main/res/layout/fragment_events.xml]: Please define all dimensions in dimension resources file.

Comment (anhanh11001) [reply app/src/main/res/layout/fragment_events.xml]: I will fix it now

Comment (anhanh11001) [reply app/src/main/res/layout/fragment_events.xml]: I fixed it, please review

File: app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt [hunk 4]
```diff
@@ -16,10 +16,11 @@ import kotlinx.android.synthetic.main.content_no_internet.view.noInternetCard
 import kotlinx.android.synthetic.main.content_no_internet.view.retry
 import kotlinx.android.synthetic.main.fragment_events.eventsNestedScrollView
 import kotlinx.android.synthetic.main.fragment_events.view.eventsRecycler
-import kotlinx.android.synthetic.main.fragment_events.view.homeScreenLL
-import kotlinx.android.synthetic.main.fragment_events.view.locationTextView
+import kotlinx.android.synthetic.main.fragment_events.view.noEventsMessage
 import kotlinx.android.synthetic.main.fragment_events.view.progressBar
 import kotlinx.android.synthetic.main.fragment_events.view.shimmerEvents
+import kotlinx.android.synthetic.main.fragment_events.view.homeScreenLL
+import kotlinx.android.synthetic.main.fragment_events.view.locationTextView
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: Why these changes?

Comment (anhanh11001) [reply app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt]: sorry for that, I will make sure I learn how to check for difference of files carefully before making a push and PR, I fixed that

File: app/src/main/res/layout/fragment_events.xml [hunk 5]
```diff
@@ -87,7 +87,12 @@
                     <FrameLayout
                         android:layout_width="match_parent"
                         android:layout_height="match_parent">
-
+                        <TextView
+                            android:id="@+id/noEventsMessage"
+                            android:layout_width="match_parent"
+                            android:layout_height="@dimen/empty_view_height"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_events.xml]: Make it wrap content

Comment (anhanh11001) [reply app/src/main/res/layout/fragment_events.xml]: @iamareebjamal please review again, I've made my change, I was having problem to make the empty-event message in the center of screen, but then I figured out that the NestedScrollView doesn't fill up the screen when there are no enough data. So I put the fillViewPort="true" in.

**Comment by anhanh11001** [comment]
@liveHarshit please review when you have the time
